django_firebase_auth/views.py

```diff
@@ -1,20 +1,23 @@
-import json
+from typing import Optional
 
-import firebase_admin
+from firebase_admin import credentials, auth, initialize_app
+from firebase_admin.auth import ExpiredIdTokenError
+from firebase_admin.exceptions import FirebaseError
 from django.contrib.auth import get_user_model
-from django.http import HttpRequest, HttpResponse
+from django.http import HttpRequest, JsonResponse
 from django.http.request import HttpHeaders
 from django.conf import settings
 from django.contrib.auth import login
-from firebase_admin import credentials, auth
-from firebase_admin.auth import ExpiredIdTokenError
-from firebase_admin.exceptions import FirebaseError
+from django.contrib.auth.models import AbstractBaseUser
 
-if settings.GOOGLE_SERVICE_ACCOUNT_FILE:
-    firebase_credentials = credentials.Certificate(settings.GOOGLE_SERVICE_ACCOUNT_FILE)
-    firebase_admin.initialize_app(firebase_credentials)
+AUTH_BACKEND = settings.DJANGO_FIREBASE_AUTH_AUTH_BACKEND
+SERVICE_ACCOUNT_FILE = settings.DJANGO_FIREBASE_AUTH_SERVICE_ACCOUNT_FILE
```

Comment:
those two settings are required so when they are not present we should fail server initialization